bundles/CoreBundle/Command/ClassesDefinitionsBuildCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Command;
+
+use Pimcore\Console\AbstractCommand;
+use Pimcore\DataObject\ClassBuilder\PHPClassWriterInterface;
+use Pimcore\DataObject\ClassBuilder\PHPFieldCollectionClassWriterInterface;
+use Pimcore\DataObject\ClassBuilder\PHPObjectBrickClassWriterInterface;
+use Pimcore\DataObject\ClassBuilder\PHPObjectBrickContainerClassWriterInterface;
+use Pimcore\Model\DataObject;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @internal
+ */
+class ClassesDefinitionsBuildCommand extends AbstractCommand
+{
+    public function __construct(
+        protected PHPClassWriterInterface $classWriter,
+        protected PHPFieldCollectionClassWriterInterface $collectionClassWriter,
+        protected PHPObjectBrickClassWriterInterface $brickClassWriter,
+        protected PHPObjectBrickContainerClassWriterInterface $brickContainerClassWriter,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('pimcore:build:classes')
```

Comment:
I think we should choose a more self explaining name here, having also `pimcore:deployment:classes-rebuild` in mind. Maybe also in the same namespace and we can also think about renaming `pimcore:deployment:classes-rebuild` and providing a fallback? WDYT?

Reply:
It is not a fallback to `pimcore:deployment:classes-rebuild`. It is a different approach, this here just builds the classes, it doesn't run database schema changes or even creation of the classes. So, the idea was to introduce the 'build' namespace for things that can happen during a eg. docker build.

Reply:
Seems like a misunderstanding 😉 I know it's for a different purpose, I just thought it would make sense to have related stuff in the same groups. I'm not sure if it makes sense to introduce another namespace for that ... we can leave that open for discussion for now.